src/types/index.ts

```diff
@@ -159,7 +159,48 @@ type TVaultsAddress = {
   [vaultAddress: string]: string | any;
 };
 
-////
+//// DAO
```

Comment:
interface -> first letter I
type -> first letter T

in this case:

TPlatformData
TGitHubUser 
etc.